.drone.yml

```diff
@@ -0,0 +1,40 @@
+build:
+  image: ubuntu:xenial
+  pull: true
+  environment:
+#     Change the following to reflect your local preferences
+    - PACKAGE=firedrake
+    - FIREDRAKE_HOME=/firedrake
+    - FIREDRAKE_GID=1000
+    - FIREDRAKE_UID=1000
+    - VIRTUALENV_PATH=/firedrake/firedrake/bin:/usr/sbin:/usr/bin:/sbin:/bin
+  commands: 
+#     Bring the image up to date
+    - apt-get -y update
+    - apt-get -y dist-upgrade
+#     Install dependencies for a single run of firedrake-install
+    - apt-get -y install sudo python-minimal python-pip
+    - pip install virtualenv
+#     Set up a non-root user with sudo access to install firedrake
+    - mkdir $FIREDRAKE_HOME
+    - groupadd -g $FIREDRAKE_GID firedrake
+    - useradd -d $FIREDRAKE_HOME -u $FIREDRAKE_UID -g $FIREDRAKE_GID firedrake
+    - chown -R firedrake.firedrake $FIREDRAKE_HOME
+#     A colon in the yaml breaks the parser; to get around this, awk is used to
+#     generate the colon at runtime
+    - awk 'BEGIN { print "firedrake ALL=NOPASSWD\x3A ALL" }' >> /etc/sudoers
+#     Become the firedrake user; change to the build directory
+    - cd $FIREDRAKE_HOME
```

Comment:
I tried it and it was non-persistent beyond the line on which the change occurs.